repository-ydb-v2/src/main/java/tech/ydb/yoj/repository/ydb/statement/FindStatement.java

```diff
@@ -4,27 +4,43 @@
 import tech.ydb.yoj.databind.schema.Schema;
 import tech.ydb.yoj.repository.db.Entity;
 import tech.ydb.yoj.repository.db.EntitySchema;
-import tech.ydb.yoj.repository.ydb.yql.YqlPredicate;
+import tech.ydb.yoj.repository.ydb.yql.YqlOrderBy;
 import tech.ydb.yoj.repository.ydb.yql.YqlStatementPart;
 
+import java.util.ArrayList;
 import java.util.Collection;
-import java.util.function.Function;
+import java.util.List;
 
 import static java.util.Comparator.comparing;
 import static java.util.stream.Collectors.joining;
 
 public class FindStatement<ENTITY extends Entity<ENTITY>, RESULT> extends PredicateStatement<Collection<? extends YqlStatementPart<?>>, ENTITY, RESULT> {
```

Comment:
naming: the naming is quite unfortunate here, we have `FindYqlStatement` (which searches by criteria) and `FindStatement` (which searches by ID). Let's maybe rename `FindStatement` to `FindByIdStatement` or something like that?

Reply:
It could produce more problem on update projects. It's could be done, but not in this track